contracts/src/BackedCommunityTokenV1.sol

```diff
@@ -0,0 +1,145 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.12;
+
+import {IBackedCommunityTokenV1} from "./interfaces/IBackedCommunityTokenV1.sol";
+import {IBackedCommunityTokenDescriptorV1} from "./interfaces/IBackedCommunityTokenDescriptorV1.sol";
+import {BackedCommunityStorageV1} from "./BackedCommunityStorageV1.sol";
+import {IERC721} from "../lib/openzeppelin-contracts/contracts/token/ERC721/IERC721.sol";
+import {ERC721SoulboundUpgradeable} from "./ERC721SoulboundUpgradeable.sol";
+
+contract BackedCommunityTokenV1 is
+    BackedCommunityStorageV1,
+    ERC721SoulboundUpgradeable,
+    IBackedCommunityTokenV1
+{
+    // ==== modifiers ====
+    modifier onlyAdmin() {
+        require(msg.sender == admin, "BackedCommunityTokenV1: not admin");
+        _;
+    }
+
+    // ==== constructor ====
+    function initialize(address _admin, address _descriptor)
+        public
+        initializer
+    {
+        __ERC721Soulbound_init("BackedCommunity", "BACKED");
+        admin = _admin;
+        descriptor = _descriptor;
+    }
+
+    // ==== state changing external functions ====
+
+    function addCategory(string memory displayName)
+        external
+        override
+        onlyAdmin
+    {
+        uint256 currentCategoryId = categoryCount++;
+        categoryIdToDisplayName[currentCategoryId] = displayName;
+    }
+
+    function addSpecialAccessory(address artContract)
+        external
+        override
+        onlyAdmin
+    {
+        uint256 currentAccessoryId = specialAccessoryCount++;
+        accessoryIdToArtContract[currentAccessoryId] = artContract;
+    }
+
+    function setCategoryScores(CategoryScoreChange[] memory changes)
```

Comment:
same comment on left on doc: do we want to have power to reset scores or should it be increase only?

Reply:
i don't see any downsides with being able to reset in the case of accidental increments? curious your thoughts though if you think it should be increase only

Reply:
yeah good point, will think on it more, but think fine to be able to reset in the case of errors?